### DISCUSSION TITLE
PPTP / IPSEC glueturn support for private VPN servers

### DISCUSSION THREAD

**Discussion by abigail68** [qdm12/gluetun#2053] [Ideas]
Why glueturn don't add support to PPTP / IPSEC protocols for private VPN servers?. They are very fast and use very few resources both on the nas and on the router that acts as a server. For many communications encryption is not the problem at all. Asus routers are very fast and convenient providing PPTP / IPSEC VPN SERVERS. Why not connect to your Asus home / holiday home / work router at full speed using PPTP / IPSEC VPN? 

A VPN is to exit through another IP as efficiently as possible. Who surfs through their own IP by adding layers and layers of encryption? It doesn't make sense.

Programmers should be like architects. You put beams on each building according to the weight it will support, not like nerds who, as the beams are virtual and don't cost money, they put 1000 tons of concrete on your mother's Netflix protecting it as if it were Fort Knox. Amazon is Amazon and Paypal is Paypal because there are no nerds behind it who, like the famous Japanese bureaucracy, axe the dear user they serve.

**Comment by ga2p** [comment]
Hello,

Can you add PPTP protocol?
Thank you so much.
